test/functional/Java8andUp/src_110_up/org/openj9/test/java/lang/Test_Class.java

```diff
@@ -446,7 +446,7 @@ public void test_getMethods_subtest6() throws Exception {
 	ClzImpl clzImpl = new ClzImpl();
 	for (Method method : clzImpl.getClass().getMethods()) {
 		for (Class<?> anInterface : method.getDeclaringClass().getInterfaces()) {
-			if (!anInterface.getName().equals("java.lang.IdentityInterface")) {
+			if (!anInterface.getName().equals("java.lang.IdentityObject")) {
```

Comment:
Can you try Keith's suggestion from last time, basically perform the opposite check. 

```
if (interface.getName.equals(interfaceWeNeed) { ...
```